packages/tools/client-debugger/client-debugger-view/src/components/AudienceView.tsx

```diff
@@ -62,6 +62,47 @@ export function AudienceView(props: AudienceViewProps): React.ReactElement {
 		};
 	}, [clientDebugger, audience, setAllAudienceMembers, setAudienceHistory]);
 
+	const audienceClientMetaData: AudienceClientMetaData[] = [];
+
+	for (const [clientId, client] of allAudienceMembers) {
+		audienceClientMetaData.push({ clientId, client });
+	}
+
+	return (
+		<_AudienceView
+			clientId={myClientId}
+			audienceClientMetaData={audienceClientMetaData}
+			onRenderAudienceMember={onRenderAudienceMember}
+			audienceHistory={audienceHistory}
+		/>
+	);
+}
+
+/**
+ * {@link _AudienceView} input props.
+ */
+export interface _AudienceViewProps {
+	clientId: string | undefined;
```

Comment:
You should actually be able to remove this property if you have this interface extend `HasClientId` - that will ensure the documentation gets captured too.